euphonic/cli/powder_map.py

```diff
@@ -87,9 +87,9 @@ def get_parser() -> ArgumentParser:
 
 
 def _get_broaden_kwargs(q_broadening: Optional[Sequence[float]] = None,
-                        q_unit: Unit = ureg('1/angstrom').units,
+                        q_unit: Unit = ureg('1/angstrom').units,  # noqa: B008
```

Comment:
I believe you might be able to do
```suggestion
                        q_unit: Unit = ureg('1/angstrom').units,  # noqa: function-call-in-default-argument
```
Which is a bit more self documenting (though in this case quite verbose)

Reply:
Surely that goes over the line width 😞

Reply:
Will leave it for now but if there was a better-flowing way to do this I'd like to.